src/common/base/NebulaKeyUtils.inl

```diff
@@ -0,0 +1,37 @@
+/* Copyright (c) 2019 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License,
+ * attached with Common Clause Condition 1.0, found in the LICENSES directory.
+ */
+
+
+namespace nebula {
+
+template<typename T>
+typename std::enable_if<std::is_integral<T>::value, int32_t>::type
+NebulaKeyUtils::encodeVersion(T v, uint8_t *encode) {
+    size_t len = folly::encodeVarint(v, encode);
+    DCHECK_GT(len, 0UL);
+    uint8_t temp;
+    // Keep the original order
+    for (size_t i = 0; i < len / 2; i++) {
```

Comment:
Why do we need to reverse the order both in encode and decode?

Reply:
> Why do we need to reverse the order both in encode and decode?

Sorry for the late reply! 
The reason：
* 300 is greater than 200, however, the numbers(ps: 200, 300) encoded by `folly::encodeVarint` does not guarantee the original order.  In order to maintain the original order, reverse the byte order.  [folly::encodeVarint principle](https://developers.google.com/protocol-buffers/docs/encoding?csw=1#varints)

* The `key` in RocksDb sorted in lexicographic order.